src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -64,6 +69,10 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 	private dirtyCells: number[] = [];
 	protected plugins = new Array<Array<Slick.Plugin<any>>>();
 
+	//list of column names to index.
+	private _columnNameToIndex: any;
```

Comment:
is there a better date type we could use here?

nit: it looks there is an extra blank line below.

Reply:
Resolved